tests/test_runtime.py

```diff
@@ -278,23 +277,22 @@ def test_template_inheritance(self):
         self.assertIn("missing includes: [+/templates/production]", str(err.exception))
 
     def test_sensitive_logging(self):
+        handler = logging.getLogger().handlers[0]
+        old_stream = handler.stream
         try:
-            _logHandler, oldLevel = init_logging(logging.DEBUG)
-            oldStream = _logHandler.stream
-            _logHandler.stream = six.StringIO()
+            handler.stream = six.StringIO()
 
             logger = logging.getLogger("unfurl")
             logger.info("test1 %s", sensitive_str("sensitive"))
 
             logger = logging.getLogger("another")
             logger.debug("test2 %s", sensitive_str("sensitive"))
 
-            logOutput = _logHandler.stream.getvalue()
-            self.assertIn(sensitive_str.redacted_str, logOutput)
-            self.assertNotIn("sensitive", logOutput)
+            log_output = handler.stream.getvalue()
+            self.assertIn(sensitive_str.redacted_str, log_output)
+            self.assertNotIn("sensitive", log_output)
         finally:
-            _logHandler.stream = oldStream
-            init_logging(oldLevel)
```

Comment:
this isn't needed anymore to resource the old log level? how does that happen now?

Reply:
We don't change log level at the beginning of the function anymore so no need to do it here.